.luacheckrc

```diff
@@ -1,6 +1,7 @@
 allow_defined = true
 exclude_files = {
 	"dist/",
+	"src/common/crypto"
```

Comment:
Are all these crypto files duplicated from IO contract? Or do they have any differences we should review

Reply:
I think they are duplicated, will replace just in case

Reply:
Looks like they were indeed already duplicated.